examples/vision/md/semisupervised_simclr.md

```diff
@@ -1,878 +1,883 @@
-the temperature's value as well (which can be done by defining it as a
-tf.Variable, and applying gradients on it). Even though this provides a good baseline
-value, in my experiments the learned temperature was somewhat lower
-than optimal, as it is optimized with respect to the contrastive loss, which is not a
-perfect proxy for representation quality.
-- **Image augmentation strength**: during pretraining stronger augmentations
-increase the difficulty of the task, however after a point too strong
-augmentations will degrade performance. During finetuning stronger
-augmentations reduce overfitting while in my experience too strong
-augmentations decrease the performance gains from pretraining. The whole data
-augmentation pipeline can be seen as an important hyperparameter of the
-algorithm, implementations of other custom image augmentation layers in Keras
-can be found in
-[this repository](https://github.com/beresandras/image-augmentation-layers-keras).
-- **Learning rate schedule**: a constant schedule is used here, but it is
-quite common in the literature to use a
-[cosine decay schedule](https://www.tensorflow.org/api_docs/python/tf/keras/experimental/CosineDecay),
-which can further improve performance.
-- **Optimizer**: Adam is used in this example, as it provides good performance
-with default parameters. SGD with momentum requires more tuning, however it
-could slightly increase performance.
-
----
-## Related works
-
-Other instance-level (image-level) contrastive learning methods:
-
-- [MoCo](https://arxiv.org/abs/1911.05722)
-([v2](https://arxiv.org/abs/2003.04297),
-[v3](https://arxiv.org/abs/2104.02057)): uses a momentum-encoder as well,
-whose weights are an exponential moving average of the target encoder
-- [SwAV](https://arxiv.org/abs/2006.09882): uses clustering instead of pairwise
-comparison
-- [BarlowTwins](https://arxiv.org/abs/2103.03230): uses a cross
-correlation-based objective instead of pairwise comparison
-
-Keras implementations of **MoCo** and **BarlowTwins** can be found in
-[this repository](https://github.com/beresandras/contrastive-classification-keras),
-which includes a Colab notebook.
-
-There is also a new line of works, which optimize a similar objective, but
-without the use of any negatives:
-
-- [BYOL](https://arxiv.org/abs/2006.07733): momentum-encoder + no negatives
-- [SimSiam](https://arxiv.org/abs/2011.10566)
-([Keras example](https://keras.io/examples/vision/simsiam/)):
-no momentum-encoder + no negatives
-
-In my experience, these methods are more brittle (they can collapse to a constant
-representation, I could not get them to work using this encoder architecture).
-Even though they are generally more dependent on the
-[model](https://generallyintelligent.ai/understanding-self-supervised-contrastive-learning.html)
-[architecture](https://arxiv.org/abs/2010.10241), they can improve
-performance at smaller batch sizes.
+
```

Comment:
There's an unnecessary blank line here. Beyond that, there seems to be a lot of lines that are marked as modified as git but that shouldn't be; any clue as to why? Some whitespace changes?

Reply:
You're right. It seems a bit strange, but in general I did not change a big deal in formatting. When using a local difftool for comparison it also does not show as many changes as here in GitHub, maybe that is some markdown GitHub issue.

I created the files with the suggested `python autogen.py add_example vision/semisupervised_simclr.py`